### ISSUE TITLE
Batch Normalization: Accelerating Deep Network Training by Reducing Internal Covariate Shift

### ISSUE THREAD

**Issue by yj-song131** [GDSC-Ewha-4th/Study-paperReview#36]
1. abstract  
* deep neural network를 학습하는 과정은 그 전 layer의 parameter가 변경됨에 따라 훈련(train) 중에 각 layer의 input이 바뀌기 때문에 굉장히 복잡하다. 이는 낮은 learning rate와 parameter 초기화를 요구하여 nonlinear한 model을 train하기 어렵게 만든다.
우리는 이 현상을 **'internal covariate shift'** 라고 하며, layer의 input을 normalize해서 문제를 해결한다. 
우리의 method는 normalization을 모델 아키텍쳐의 일부로 만들고 각 미니 배치에 대해 normalization을 수행하는 것에서 강점을 얻는다. 또한 regularizer 역할을 하기 때문에 어떤 경우에는 dropout이 필요하지 않다. 최첨단 이미지 분류 모델에 적용된 batch normalization은 14배 적은 training 단계로 같은 accuracy를 보이고 원래 모델을 상당한 차이로 능가했다. 그리고 batch normalized된 network의 앙상블을 사용해 ImageNet 분류에 대해 4.9% top-5 validation error(4.8%의 test error)를 달성해 인간의 정확도를 능가하였다.
2. Introduction
* **SGD**(Stochastic gradient descent)는 training deep networks의 효율을 높여주었다. 하지만 세심한 hyper parameter 튜닝을 필요로 한다. parameter의 작은 변화들이 모델에 큰 영향을 미치기 때문이다.
* 이전까지는 Gradinet Vanishing, saturation problem의 해결방법으로 ReLU와 small learning rates 방법이 제안되어 왔다. 
이 논문에서는 다른 해결방법으로 batch Normalization을 제안한다. batch norm은 layer input data의 평균과 분산을 고정시켜 데이터가 일정한 분포를 가지게 한다. 또한 parameter의 초기값과 크기에 대한 gradient의 의존도를 줄여 gradient flow에 긍정적인 영향을 준다. 이로 인해 더 높은 learning rate 사용이 가능해지고 dropout의 필요성이 줄어든다
3. Conclusion
* 'internal covariate shift'를 완화시켜서 하이퍼 파라미터 설정과 초기 파라미터 값 설정에서 자유로워지고, 자체적으로 regularization 효과가 있어 dropout을 사용하지 않아도 된다.

**Comment by minha62** [comment]
14배 적은 training 단계로 좋은 성능을 보였다니..!
좋은 내용 감사합니다:)

**Comment by penguin1109** [comment]
Batch Normalization이 모델 학습 할 때에 초기 init weight의 값에 크게 영향을 받지 않게 한다는 장점이 있기도 하면서 간혹 semantic 특징을 잃게 한다고 해서 이유가 궁금하네요!

**Comment by Jii111** [comment]
저도 모델 적용할 때 batch normalization을 적용해봐야겠어요. 좋은 내용 공유 감사합니다!
